docs/0.1/grid_on_splinter.md

```diff
@@ -0,0 +1,459 @@
+   --owners $(cat /root/.splinter/keys/gridd.pub) \
+   -k gridd \
+   -U 'http://splinterd-alpha:8085' \
+   --service-id $CIRCUIT_ID::gsAA
+   ```
+
+7. Grant the appropriate contract namespace permissions.
+
+   ```
+   root@scabbard-cli-alpha:/# scabbard perm 5b7349 sawtooth_xo --read --write \
+   -k gridd \
+   -U 'http://splinterd-alpha:8085' \
+   --service-id $CIRCUIT_ID::gsAA
+   ```
+
+
+8. Open a new terminal and connect to the `scabbard-cli-beta` container and add
+   the circuit ID environment variable
+   ```
+   $ docker-compose -f examples/splinter/docker-compose.yaml run scabbard-cli-beta bash
+   root@scabbard-cli-beta:/#
+   ```
+   ```
+   root@scabbard-cli-beta:/# export CIRCUIT_ID=01234-ABCDE
+   ```
+
+9. List all uploaded smart contracts.
+
+   ```
+   root@scabbard-cli-beta:/# scabbard contract list -U 'http://splinterd-beta:8085' --service-id $CIRCUIT_ID::gsBB
+   NAME        VERSIONS OWNERS
+   grid_product 1.0      <gridd-alpha public key>
+   pike         0.1      <gridd-alpha public key>
+   sawtooth_xo  1.0      <gridd-alpha public key>
+   ```
+
+10. Display the xo smart contract.
+
+   ```
+   root@scabbard-cli-beta:/# scabbard contract show sawtooth_xo:1.0 -U 'http://splinterd-beta:8085' --service-id $CIRCUIT_ID::gsBB
+   sawtooth_xo 1.0
+     inputs:
+     - 5b7349
+     outputs:
+     - 5b7349
+     creator: <gridd-alpha public key>
+   ```
+
+## Demonstrate Circuit Scope
+
+If a node is not a part of a circuit, that node cannot access information about
+that circuit or any transactions that occur on that circuit.
+
+Use the following steps to demonstrate that the third node in the network
+(gamma-node-000) cannot see the circuit between alpha and beta, even when it
+participates in a new multi-party circuit with those nodes.
+
+1. Connect to the splinterd-gamma Docker container. You will use this container
+   to run Splinter commands on gamma-node-000.
+
+   ```
+   $ docker-compose -f examples/splinter/docker-compose.yaml exec splinterd-gamma bash
+   root@splinterd-gamma:/#
+   ```
+
+2. Verify that splinterd-gamma does not see any circuits.
+   ```
+   root@splinterd-gamma:/# splinter circuit list --url http://splinterd-gamma:8085
+   ID MANAGEMENT MEMBERS
+   ```
+
+Final note: Splinter strictly enforces privacy for all information on a
+circuit, including participants, available smart contracts, and transactions
+performed by the participants using those smart contracts.
+
+For example, if gamma creates a circuit with alpha and a separate circuit with
+beta, then uploads the XO smart contract and plays a tic-tac-toe game with
+alpha, the xo list command on gamma will show only the gamma-alpha game. Even
+though alpha and beta are using the same XO smart contract, their game moves
+(smart contract transactions) remain private to their two-party circuit.
+
+## For More Information
+- [Hyperledger Grid documentation](https://grid.hyperledger.org/docs/grid/nightly/master/introduction.html)
```

Comment:
Since we're already "in" the Grid docs, replace this with a link to the Splinter documentation:

`[Splinter documentation](https://www.splinter.dev/docs/)`